test/com/minseungkwon/datastructure/Graph/BinaryTreeTest.java

```diff
@@ -319,4 +322,20 @@ public void test_findCommonAncestorUsingPostOrder() {
         assertEquals(7, bt.findCommonAncestorUsingPostOrder(7, 9).data);
         assertEquals(null, bt.findCommonAncestorUsingPostOrder(7, 11));
     }
+
+    @Test
+    public void test_findAllCombination(){
+        Integer[] arr = makeArrayForTree(5);
+        BinaryTree<Integer> bt = new BinaryTree<>();
+        bt.makeTree(arr);
+        ArrayList<LinkedList<Node>> result = bt.findAllCombination(bt.getRoot());
+        StringBuilder s = new StringBuilder();
+        for(LinkedList<Node> ll : result){
```

Comment:
Instead of appending all the result into a single string, I think you can also specify the results in the list and compare them one by one or at once.

Reply:
Okie